hangupsbot/plugins/default.py

```diff
@@ -48,7 +48,12 @@ def echo(bot, event, *args):
 
         yield from command.run(bot, event, *["convecho", "id:" + convid, _text])
 
-
+def admins(bot,event,*args):
+    """Get a list of bot admins"""
+    admins = ' <br>'.join([bot.get_hangups_user(uid).full_name for uid in bot.get_config_option('admins')])
```

Comment:
since we're on the theme about python programming, python list comprehensions are extremely powerful and compact - this can be rewritten as the following for readability (which python prizes):
```
admins = '<br>'.join([ bot.get_hangups_user(uid).full_name
                       for uid in bot.get_config_option('admins') ])
```
and then the bot user can be excluded by adding the following IF statement directly into the list comprehension:
```
admins = '<br>'.join([ bot.get_hangups_user(uid).full_name
                       for uid in bot.get_config_option('admins')
                       if uid != bot.user_self()["chat_id"] ])
```
disclaimer: haven't tested this, but should be similar to its final form

related documentation:
* http://stackoverflow.com/a/4406777
* https://github.com/hangoutsbot/hangoutsbot/wiki/Functions-for-Plugin-and-Bot-Developers